openedx/core/djangoapps/verified_track_content/management/commands/swap_from_auto_track_cohort_pilot.py

```diff
@@ -0,0 +1,268 @@
+from contentstore.course_group_config import GroupConfiguration
+from django.conf import settings
+from course_modes.models import CourseMode
+from django.core.management.base import BaseCommand, CommandError
+
+from openedx.core.djangoapps.course_groups.cohorts import CourseCohort
+from openedx.core.djangoapps.course_groups.models import (CourseUserGroup, CourseUserGroupPartitionGroup)
+from openedx.core.djangoapps.verified_track_content.models import (
+    MigrateVerifiedTrackCohortsSetting,
+    VerifiedTrackCohortedCourse
+)
+
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.django import modulestore
+from xmodule.partitions.partitions import ENROLLMENT_TRACK_PARTITION_ID
+from xmodule.partitions.partitions_service import PartitionService
+
+
+class Command(BaseCommand):
+    """
+    Migrates a course's xblock's group_access from Verified Track Cohorts to Enrollment Tracks
+    """
+
+    def handle(self, *args, **options):
+
+        errors = []
+
+        module_store = modulestore()
+
+        print "Starting Swap from Auto Track Cohort Pilot command"
+
+        verified_track_cohorts_setting = self._latest_settings()
+
+        if not verified_track_cohorts_setting:
+            raise CommandError("No MigrateVerifiedTrackCohortsSetting found")
+
+        old_course_key = verified_track_cohorts_setting.old_course_key
+        rerun_course_key = verified_track_cohorts_setting.rerun_course_key
+        audit_cohort_names = verified_track_cohorts_setting.get_audit_cohort_names()
+
+        # Verify that the MigrateVerifiedTrackCohortsSetting has all required fields
+        if not old_course_key:
+            raise CommandError("No old_course_key set for MigrateVerifiedTrackCohortsSetting with ID: '%s'"
+                               % verified_track_cohorts_setting.id)
+
+        if not rerun_course_key:
+            raise CommandError("No rerun_course_key set for MigrateVerifiedTrackCohortsSetting with ID: '%s'"
+                               % verified_track_cohorts_setting.id)
+
+        if not audit_cohort_names:
+            raise CommandError("No audit_cohort_names set for MigrateVerifiedTrackCohortsSetting with ID: '%s'"
+                               % verified_track_cohorts_setting.id)
+
+        print "Running for MigrateVerifiedTrackCohortsSetting with ID='%s'" % verified_track_cohorts_setting.id
```

Comment:
Honestly, I was pretty confused about what this ID was. It might be better to just print the rerun course ID.